xbmc/cores/VideoPlayer/DVDInputStreams/DVDFactoryInputStream.cpp

```diff
@@ -150,6 +150,9 @@ std::shared_ptr<CDVDInputStream> CDVDFactoryInputStream::CreateInputStream(IVide
       }
     }
 
+    if (URIUtils::IsProtocol(finalFileitem.GetPath(), "pvr"))
```

Comment:
Mhm not sure if `GetPath()` is correct here.
You probably need to check the `DynPath`.
@ksooo probably knows?

Reply:
The intention was to pick up streams that were not already detected as DVDSTREAM_TYPE_PVRMANAGER. From testing they always appeared to preset satisfying the above condition.

Reply:
path == item identity (must not change during item instance' lifetime)
dynpath == "playback url" (can change at any time, thus the name)

most of the time path == dynpath

Examples: 
* pvr.hts (own demuxer) recordings/channels. path = dynpath = "pvr://..."
* pvr.iptvsimple (no own demuxer): recordings/channels. path = "püvr://...", dynpath = "http(s)://..."